hud-data/quotehud.json

```
@@ -1,11 +1,73 @@
 {
   "name": "QuoteHUD",
-  "author": "QuoteStrife",
+  "author": "Qüöte | quotestrife",
+  "contributors": [
+    "Broesel-tf",
+    "DETENOG"
+  ],
+  "releaseDate": "2019-05-25",
+  "parent": "bear-hud",
+  "distinguishedFromParent": true,
+  "resources": [
+    "quotehud-banner",
+    "quote1",
+    "quote2",
+    "quote3",
+    "quote4",
+    "quote5",
+    "quote6",
+    "quote7"
+  ],
+  "flags": [
+    "fonts",
+    "menus",
+    "scoreboards",
+    "crosshairs",
+    "materials"
+  ],
+  "tags": [
+    "minimal",
+    "cartoony"
+  ],
+  "repo": "https://github.com/Silk-OT/QuoteHUD",
+  "hash": "c33c8d51ed265dc12170df77885c45f782cc11b6",
+  "prerelease": false,
   "social": {
-    "album": "https://imgur.com/a/E58DNqI"
+    "steam_profile": "76561198166167645",
+    "steam_group": "quotestrife",
+    "twitter": "SillySilk_",
+    "bsky": "quotesilk@bsky.social",
+    "discord": "quotestrife",
```

```suggestion
    "discord": "quotestrife#0",
```

For reference to what mastercoms was referring to @Silk-OT